pkg/client/flags.go

```diff
@@ -38,6 +39,7 @@ func BindClientConfigFlags(flags FlagSet, config *Config) {
 	flags.StringVar(&config.CertFile, "client_certificate", config.CertFile, "Path to a client key file for TLS.")
 	flags.StringVar(&config.KeyFile, "client_key", config.KeyFile, "Path to a client key file for TLS.")
 	flags.StringVar(&config.CAFile, "certificate_authority", config.CAFile, "Path to a cert. file for the certificate authority.")
+	flags.Float32Var(&config.QPS, "max_qps", config.QPS, "Maximum number of queries per second that could be issued by this client.")
```

Comment:
Take a look at our rate limiter - https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/util/throttle.go#L47

It actually takes a second parameter that controls bursting. Might want to take that here, too.